src/components/dashboard/forms/items/NewItemFormTags.js

```diff
@@ -0,0 +1,99 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { bindActionCreators } from 'redux'
+import actions from 'actions'
+import NewItemHoc from './NewItemHocStep'
+import Translate from 'components/translate/Translate'
+import Grid from '@material-ui/core/Grid'
+import Autocomplete from 'components/common/autocomplete'
+import { getTags } from 'services/adex-node/actions'
+
+const noTagsErrMsg = 'ERR_REQUIRED_FIELD'
+
+class NewItemFormTags extends Component {
+    componentWillMount() {
+        this.props.validate('tags', {isValid: this.props.item.meta.tags && this.props.item.meta.tags.length > 0, err: {msg: noTagsErrMsg}})                             
+            getTags({authSig: this.props.account._authSig})
```

Comment:
authSig is not needed anymore as the node requester gets the sig from the store. You can remove it where you found it.